src/PlotlyPanel.tsx

```diff
@@ -155,11 +152,17 @@ export const PlotlyPanel: React.FC<Props> = (props) => {
             plotData.length === 0 || !plotData.find((d) => d.y?.length) ? [{ text: 'No data', showarrow: false }] : [],
           ...getLayout(theme, options, plotData, axisLabels),
         }}
-        config={{ displayModeBar: false }}
+        config={getConfig(options)}
         onClick={handlePlotClick}
+        onRelayout={handlePlotRelayout}
       />
       {menu.show && (
-        <ContextMenu items={menu.items} x={menu.x} y={menu.y} onClose={() => setMenu({ ...menu, show: false })} />
+        <ContextMenu
```

Comment:
The context menu API changed upgrading to Grafana 9, so the way supply the items changed. This fixed 'data links' not working correctly.